docs/dev/dependencies.md

```diff
@@ -4,12 +4,23 @@
 
 The following dependencies must be installed on your system before you can build the installer.
 
-### Fedora, CentOS, RHEL
+### Fedora
 
 ```sh
 sudo yum install golang-bin gcc-c++
 ```
 
+### CentOS, RHEL
+
+You need to install [go-toolset-1.10](https://access.redhat.com/documentation/en-us/red_hat_developer_tools/2018.4/html/using_go_toolset/).
+
+```sh
+sudo yum-config-manager --enable rhel-7-server-devtools-rpms
+sudo yum-config-manager --enable rhel-server-rhscl-8-rpms
+sudo yum install -y go-toolset-1.10-golang-bin gcc-c++
+scl enable go-toolset-1.10 bash
+```
+
 If you need support for [libvirt destroy](libvirt-howto.md#cleanup), you should also install `libvirt-devel`.
```

Comment:
This line applies to Fedora as well.  I'm not sure how to restructure your suggestion to preserve that, aside from copy/pasting to it shows up in both sections.